packages/tracing/src/browser/metrics.ts

```diff
@@ -207,6 +208,13 @@ export class MetricsInstrumentation {
 
         transaction.setTag('lcp.size', this._lcpEntry.size);
       }
+
+      if (this._clsEntry) {
+        logger.log('[Measurements] Adding CLS Data');
+        transaction.setData('measurements.cls', {
```

Comment:
@jaketlarson Thanks for bringing this back up. I'm fine with this addition. 👍 

We'll need a minor change on how we add the CLS sources. Rather than `transaction.setData()`, it'll be preferred that `transaction.setTag()` be used (see how `lcp.element` tag is constructed above in this file):

```js
this._clsEntry.sources.map((source, index) => transaction.setTag(`cls.source.${index + 1}`, htmlTreeAsString(source.node)));
```

Reply:
Fixed in [`e67b8bd` (#3734)](https://github.com/getsentry/sentry-javascript/pull/3734/commits/e67b8bdcd5b764a6a78d859ab9c528abe61a932a)